Microsoft.Toolkit.Uwp.Services/Core/DataProviderBase{TConfig,TSchema}.cs

```diff
@@ -23,7 +21,7 @@ namespace Microsoft.Toolkit.Uwp.Services
     /// <typeparam name="TSchema">Strong typed object to parse the response items into.</typeparam>
     [Obsolete("This class is being deprecated. Please use the .NET Standard Library counterpart found in Microsoft.Toolkit.Services.")]
```

Comment:
this warning to be changed to the parsers namespace

Reply:
This class is still in the Services namespace.